src/test/java/com/artipie/rpm/http/RpmSliceITCase.java

```diff
@@ -37,6 +39,7 @@
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
+@DisabledOnOs(OS.WINDOWS)
```

Comment:
Same question here.

Reply:
windows github actions do not have docker, that's why these ITs are disabled on win os